projects/extension/src/background/ConnectionManagerWithHealth.ts

```diff
@@ -50,6 +50,11 @@ export interface ChainInfo<SandboxId> {
    * The identifier for the sandbox that has received the message that requests to add a chain.
    */
   sandboxId: SandboxId
+
+  /**
+   * Last block of the chain
+   */
+  lastBlock: number
```

Comment:
```suggestion
   * Height of the current best block of the chain
   */
  latestBestBlock: number
```

"last" implies that there won't be any other block in the future.